x/cork/types/cork.go

```diff
@@ -17,6 +18,16 @@ func (c *Cork) InvalidationScope() tmbytes.HexBytes {
 		)).Bytes()
 }
 
+func (c *Cork) IDHash(blockHeight uint64) []byte {
+	blockHeightBytes := sdk.Uint64ToBigEndian(blockHeight)
+
+	return crypto.Keccak256Hash(
+		bytes.Join(
+			[][]byte{blockHeightBytes, []byte(c.TargetContractAddress), c.EncodedContractCall},
```

Comment:
The point was raised in chat that we may want to convert the target contract address to a `common.Address` before converting to bytes

Reply:
yea, I saw that comment by Eric. Waiting on him to give the final call

Reply:
Yes, we should make that change.